draft-ietf-ice-rfc5245bis.xml

```diff
@@ -2840,14 +2843,15 @@ agent continue to listen for STUN requests and continue to generate
 triggered checks for a media stream, even once processing for that
 stream completes. The rules in this section describe when it is safe
 for an agent to cease sending or receiving checks on a candidate that
-was not selected by ICE, and then free the candidate.
+did not become a selected candidate (is not associated with a selected pair), 
+and then free the candidat.
 </t>	
 	
 <t>
 Once a checklist has reached the Completed state, the agent SHOULD wait an
 additional three seconds, and then it can cease responding to checks or
 generating triggered checks on all local candidates other than
-the ones in the selected candidate pairs (one for each component).
+the ones that became selected candidates.
```

Comment:
I don't recall the term "selected candidate" being used before. Should this be added to "Terminology"?

Reply:
It should be there already.